.github/workflows/submodule_update.yml

```diff
@@ -0,0 +1,38 @@
+name: submodule_update
+
+on:
+  push:
+    branches: 
+      - 'master'
+      - '2.10'
+      - 'foxzi_update_submodule_workflow'
+
+jobs:
+  tarantool:
+    name: Update submodule in EE repository
+    runs-on: ubuntu-20.04-self-hosted
+    env:
+      SUBMODULE: 'tarantool'
+      REPOSITORY: 'tarantool/tarantool-ee'
+      FEATURE_BRANCH: 'TarantoolBot/update-tarantool-${{ github.ref_name }}'
+      CHECKOUT_BRANCH: master
+      PR_AGAINST_BRANCH: master
```

Comment:
Please add the following line:
```
PR_TITLE_PREFIX: ${{ github.ref_name == 'master' && '' ||
  '[${{ github.ref_name }}] ' }}
```

Reply:
It's doesn't work but it is good idea. I and @NickVolynkin fix it.

Reply:
Ok, your variant is ok for me.